server/routes/surveyRoutes.js

```diff
@@ -61,13 +68,13 @@ module.exports = app => {
       title,
       subject,
       body,
-      recipients: recipients.split(',').map(email => ({ email: email.trim() })),
+      recipients: Mailer.getSurveyListFromEmailString(recipients),
```

Comment:
Splitting the string and building an email object all inside this object definition was too confusing to me, so I added a helper function to the Mailer service that does it. Just a little cleaner, as it is a named function that actually describes what is happening here `getRecipientListFromEmailString`.

Now it's easy to tell it's getting a recipient list from  an email string.